### PR TITLE
Updated image_key documentation

### PR DESCRIPTION
**Pull request by sanjanag**
Screenshot 2024-03-08 at 12 01 50 PM
Screenshot 2024-03-08 at 12 04 13 PM

### CODE REVIEW

File: cleanlab/datalab/datalab.py [hunk 1]
```diff
@@ -92,9 +92,10 @@ class Datalab:
         The name of the label column in the dataset.
 
     image_key : str, optional
-        Optional key that can be specified for image datasets to point to the field containing the actual images themselves.
-        If specified, additional image-specific issue types can be detected in the dataset.
-        See the CleanVision package `documentation <https://cleanvision.readthedocs.io/en/latest/>`_ for descriptions of these image-specific issue types.
+        Name of the image column in the dataset containing the PIL image objects.
+        It is an optional argument that can be specified to check for additional image-specific issues in the dataset. See the `CleanVision package <https://github.com/cleanlab/cleanvision?tab=readme-ov-file#clean-your-data-for-better-computer-vision>`_ for descriptions of these image-specific issue types.
+        Currently, this argument is only supported for "datasets.Dataset" format.
```
Comment (jwmueller) [inline cleanlab/datalab/datalab.py:97]:
```suggestion
        Optional key that can be specified for image datasets to point to the field (column) containing the actual images themselves (as PIL objects).
        If specified, additional image-specific issue types will be checked for in the dataset.
        See the `CleanVision package <https://github.com/cleanlab/cleanvision?tab=readme-ov-file#clean-your-data-for-better-computer-vision>`_ for descriptions of these image-specific issue types.
        Currently, this argument is only supported for data formatted as a Hugging Face ``datasets.Dataset`` object.
```

File: docs/source/cleanlab/datalab/guide/issue_type_description.rst [hunk 2]
```diff
@@ -157,9 +157,10 @@ Class imbalance in a dataset can lead to subpar model performance for the under-
 Image-specific Issues
 ---------------------
 
-For image datasets which are properly specified as such, Datalab can detect additional types of image-specific issues (if the necessary optional dependencies are installed).
-Specifically, low-quality images which are too: dark/bright, blurry, low information, abnormally sized, etc.
-Descriptions of these image-specific issues are provided in the `CleanVision package <https://github.com/cleanlab/cleanvision>`_ and its documentation.
+Datalab can identify image-specific issues in datasets, such as images that are excessively dark or bright, blurry, lack detail, or have unusual sizes.
+To detect these issues, simply specify the `image_key` argument in :py:meth:`~cleanlab.datalab.datalab.Datalab`, indicating the image column name in your dataset.
+This functionality currently works only with Huggingface datasets. You can convert other local dataset formats into Huggingface dataset by following `this guide <https://huggingface.co/docs/datasets/en/loading>`_.
```
Comment (jwmueller) [inline docs/source/cleanlab/datalab/guide/issue_type_description.rst:162]:
```suggestion
This functionality currently works only with Hugging Face datasets. You can convert other local dataset formats into a Hugging Face dataset by following `this guide <https://huggingface.co/docs/datasets/en/loading>`_.
```

**Review by jwmueller — APPROVED** [review]
had a couple suggestions but otherwise looks good to merge

**Review by elisno — APPROVED** [review]
LGTM!
Don't merge this until these have been merged, in the following order:

1. #1051 
2. #1052

**Pull request merged**